lib/client/dom.js

```diff
@@ -883,6 +883,14 @@ var CloudCmd, Util, DOM, CloudFunc;
                 var current     = currentFile || this.getCurrentFile();
                     
                 current.classList.toggle(SELECTED_FILE);
+
+                // OSC_CUSTOM_CODE There is a bug where deselection is delayed, this is because the "selected-file"
+                // tag is successfully removed, but not the "current-file" tag. This removes the "current-file" tag
+                // if a file is deselected.
+                // https://github.com/AweSim-OSC/osc-fileexplorer/issues/59
+                if (current.classList.contains(CURRENT_FILE) && !current.classList.contains(SELECTED_FILE)){
+                    current.classList.remove(CURRENT_FILE);
+                }
```

Comment:
This bug was caused by our changes to the CSS which make the "current-file" and "selected-file" share the same css styles.

Previously, the "current-file" was denoted by a border around the row, and the "selected-file" was a change to the text color. There's no way to differentiate now.

This only removes the "current-file" class from a file after it is deselected, which I think is ok. If the user wants to edit or view a file the "current-file" class will be reattached as soon as they right click on it.